src/main/app/src/Components/Landscape/Map/MapRelation/MapRelation.tsx

```diff
@@ -36,28 +37,27 @@ const MapRelation: React.FC<Props> = ({ source, target, relation, locateItem })
         <br />
         <Typography variant={'h6'}>Source</Typography>
         <div>
-          <Button
-            fullWidth={true}
-            key={source.fullyQualifiedIdentifier}
+          <IconButton
             onClick={() => {
               locateItem(source.fullyQualifiedIdentifier);
             }}
           >
```

Comment:
I don't know if this is exactly the position that breaks it, but the images look odd in the new icons:
<img width="294" alt="sidebar4" src="https://user-images.githubusercontent.com/37021266/104856416-d7d0c980-5912-11eb-9316-e709e8227ebf.png">

Reply:
This happens only in the sidebar, not on the map. This is probably not the correct location in the code!

Reply:
you mean we should send the item identifier (or group, relation) to the map component instead? and then the map would locate the target? yes, could be done 👍

Reply:
I don't really know what you mean. I am just pointing out that the redis image is compressed on the sidebar. 

I was just trying to locate what code change broke it!

Reply:
I thought you suggested to change a software design flaw 😄

Reply:
Haha, that could be me. But in this case I lack the overview on how the components work. But since you suggested it yourself: go ahead! 😅

Reply:
#380 is the software design flaw 😅